fvm/evm/stdlib/contract.cdc

```diff
@@ -56,8 +56,8 @@ contract EVM {
         gasConsumed: UInt64,
         // if transaction was a deployment contains a newly deployed contract address
         contractAddress: String,
```

Comment:
While at it, maybe the contract address could also be improved and and changed to `Address`?

Reply:
I'll leave it for another PR, there was some discussion in the past by the community for checking deployment by comparing this value to empty, I need to check those comments first.